### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by bill7pearl**
# In this PR I have:
- Used ES6 syntax.
- Divided my code into modules and saved each module in a separate file.
- Kept all functionalities of the app without errors.
- Refactored my methods and functions to arrow functions.
- Used let and const in a correct way.
- Displayed the current date below the navigation bar using Luxon library.

### Please note that we are facing problems when using Luxon library locally, it seems there is a bug with the original downloaded file, this issue is happening to every Micronaut I know, that's why we've used CDN instead and it's working fine.

### CODE REVIEW

File: modules/display-books.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBookStorage(book) {
+    const books = BooksLocalStorage.getBookStorage();
```
Comment (m05e5) [inline modules/display-books.js]: - [x] Nice work. As part of the requirements, kindly change all your classes and methods to arrow functions

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @bill7pearl , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [x] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/display-books.js (hunk 1 above)
Comment (bill7pearl) [reply modules/display-books.js]: Done, thanks for pointing it out

**Review by uwadonat — APPROVED** [review]
## Hi @bill7pearl,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
approval


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**